src/auction.js

```diff
@@ -740,6 +740,19 @@ function getPreparedBidForAuction(bid, {index = auctionManager.index} = {}) {
     bid.renderer.setRender(renderer.render);
   }
 
+  if (bid.rendererurl) {
```

Comment:
I think this should be done where you updated `ortbConverter ` - since it only affects the PBS adapter, there's no reason to include the logic in core. It would also remove the need to update `video.js`.

Reply:
updated to move into ortbconverter